packages/ui/src/components/Transactions/Transaction.tsx

```diff
@@ -141,7 +141,7 @@ export default styled(Transaction)(
     max-height: 1.3125rem;
     left: 29px;
     top: 19px;
-    border-radius: 0 0.9375rem 0.9375rem 0.9375rem;
+    border-radius: ${theme.custom.borderRadius};
```

Comment:
This breaks the effect of having the top left not having any in the top left. Not sure if this suggestion works, we we could try

```suggestion
    border-radius: 0  ${theme.custom.borderRadius}  ${theme.custom.borderRadius} ${theme.custom.borderRadius};
```

Reply:
I think the new design doesn't have that view for the badge. So I think we can consider it as a deprecated and keep the new design

Reply:
this is not implementing the new design though, we're in the old, changing border-radius. Now if that's something you've strong opinion about, because you don't like it, I can let it go.